kubernetes/ansible/k8s-remove.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: "Uninstall kubernetes modules"
+  hosts: "{{ override_host | default('masters') }}"
+  gather_facts: "{{ gathering_host_info | default('true') | bool == true }}"
+
+  # Define default values for variables that are common between roles
+  vars:
+    client_tool: kubectl
+
+  tasks:
+    - name: "Execute k8s_dashboard uninstall role"
+      import_role:
+        name: 'k8s_dashboard'
+      vars:
+        - remove: true
+      tags: [k8s_dashboard]
+
+    - name: "Execute Ingress Controller uninstall role"
+      import_role:
+        name: 'ingress'
+      vars:
+        - remove: true
+      tags: [always,ingress]
+
+    # k8s_config doesn't have a remove task.
+    # - name: "Execute k8s_config uninstall role"
+    #   import_role:
+    #     name: 'k8s_config'
+    #   vars:
+    #     - remove: true
+    #   tags: [always,k8s_config]
```

Comment:
To be removed if not used

Reply:
Removed it as that role doesn't have remove implemented.